assets/app/views/_volumes.html

```diff
@@ -40,11 +52,38 @@
     </div>
     <div ng-if="volume.gitRepo">
       <dt>Type:</dt>
-      <dd>git repo</dd>
+      <dd>
+        git repo
+       <span class="small text-muted">(a volume that is pulled from git when the pod is created)</span>
+      </dd>
       <dt>Repository:</dt>
       <dd>{{volume.gitRepo.repository}}</dd>
       <dt ng-if-start="volume.gitRepo.revision">Revision:</dt>
       <dd ng-if-end>{{volume.gitRepo.revision}}</dd>
     </div>
+    <div ng-if="volume.downwardAPI">
+      <dt>Type:</dt>
+      <dd>
+        downward API
+        <span class="small text-muted">(a volume populated by information about the pod)</span>
+       </dd>
+      <div ng-repeat="item in volume.downwardAPI.items">
+        <dt>Volume file:</dt>
+        <dd>{{item.fieldRef.fieldPath}}&#8201;&#8594;&#8201;{{item.path}}</dd>
+      </div>
+    </div>
+    <div ng-if="volume.configMap">
+      <dt>Type:</dt>
+      <dd>
+        config map
+        <span class="small text-muted">(a volume populated by a config map)</span>
+       </dd>
+      <dt>Name:</dt>
+      <dd>{{volume.configMap.name}}</dd>
+      <div ng-repeat="item in volume.configMap.items">
+        <dt>Volume file:</dt>
+        <dd>{{item.path}}&#8201;&#8594;&#8201;{{item.key}}</dd>
```

Comment:
hmm the problem with this is that I don't necessarily understand that the value being shown for item.key is that key of the config map, maybe

**Key to File:** foo &#8201;&#8594;&#8201; /some/path

or something like that